doc/neps/nep-0016-abstract-array.rst

```diff
@@ -3,16 +3,16 @@ NEP 16 — An abstract base class for identifying "duck arrays"
 =============================================================
 
 :Author: Nathaniel J. Smith <njs@pobox.com>
-:Status: Withdrawn
+:Status: Superseded
```

Comment:
I thought about using "Superceded" here but ended up deciding against it.

The NEP build tool wants a "Superceded" NEP to be replaced by a NEP marked as "Replaces", but I don't think this entirely makes sense, since NEP 22 has a broader scope than NEP 16, and isn't a concrete implementation proposal.